client/src/components/subscription/subscription.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Subscription = ({ details }) => {
+  const { name, nickname, dueDate, reminderDays, amount } = details;
+
+  return (
+    <ul className="subscription-details">
+      <li>Name: {name}</li>
+      <li>Nickname: {nickname}</li>
+      <li>DueDate: {dueDate}</li>
+      <li>ReminderDays: {reminderDays}</li>
```

Comment:
should these have spaces?

```suggestion
      <li>Due Date: {dueDate}</li>
      <li>Reminder Days: {reminderDays}</li>
```